docs/user/installation.rst

```diff
@@ -4,50 +4,239 @@
 Installation
 ============
 
-Lasagne currently requires Python 2.7 or 3.4 to run. Some of Lasagne's
-dependencies such as ``numpy`` may require a C compiler to install.
+Lasagne has a couple of prerequisites that need to be installed first, but it
+is not very picky about versions. The single exception is Theano: Due to its
+tight coupling to Theano, you will have to install a recent version of Theano
+(usually more recent than the latest official release!) fitting the version of
+Lasagne you choose to install.
 
-For most installations, it is recommended to install Lasagne and its
-dependencies inside a virtualenv or a conda environment. The
-following commands assume that you have your environment active.
+Most of the instructions below assume you are running a Linux or Mac system;
+please do not hesitate to suggest instructions for Windows via the *Edit on
+GitHub* link on the top right!
 
-You can either install from PyPI or install from source, such as from
-a Git clone.
+If you run into any trouble, please check the `Theano installation instructions
+<http://deeplearning.net/software/theano/install.html>`_ which cover installing
+the prerequisites for a range of operating systems, or ask for help on `our
+mailing list <https://groups.google.com/d/forum/lasagne-users>`_.
 
-Install from PyPI
-=================
 
-.. note:: Lasagne hasn't been released as of now.  The only way to
-          install it for now is from source.
+Prerequisites
+=============
 
-To install release 0.1 of Lasagne from PyPI, run the following
-commands:
+Python
+------
+
+Lasagne currently requires Python 2.7 or 3.4 to run. Please install Python via
+the package manager of your operating system if they are not included already.
+Use Python 2.7 if you're in doubt; it is still more widespread than Python 3.
+
+C compiler
+----------
+
+Theano requires a working C compiler, and numpy/scipy require a compiler as
+well if you install them via ``pip``. On Linux, the default compiler is usually
+``gcc``, and on Mac OS, it's ``clang``. Again, please install them via the
+package manager of your operating system.
+
```

Comment:
If we want to go for maximal user friendliness maybe we could have some Mac / Ubuntu instructions here as well (that combo should cover the largest part of our user base, probably). Just an idea, please don't feel obliged to add this in this PR. It's already quite comprehensive :)